packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -67,13 +67,16 @@ export class FASTAccordion extends FASTElement {
         this.$emit("change", this.activeid);
     };
 
-    private findExpandedItem(): FASTAccordionItem | null {
+    private findExpandedItem(): FASTAccordionItem | Element {
         for (let item: number = 0; item < this._accordionItems.length; item++) {
-            if (this._accordionItems[item].getAttribute("expanded") === "true") {
-                return (this._accordionItems[item] as unknown) as FASTAccordionItem;
+            if (
+                ((this._accordionItems[item] as unknown) as FASTAccordionItem)
+                    .expanded === true
+            ) {
+                return this._accordionItems[item];
             }
         }
-        return null;
+        return this._accordionItems[0];
     }
```

Comment:
Is a for-loop needed here for performance? This function could be condensed:

```ts
private findExpandedItem(): FASTAccordionItem | Element {
    return (
        this._accordionItems.find((item: FASTAccordionItem) => item.expanded) ??
        this._accordionItems[0]
    );
}
```

Reply:
I kept it relative to not making too many changes, but it's a good improvement.